src/dashboard/cd-dashboard-stats.vue

```diff
@@ -17,17 +17,25 @@
 
 <style scoped lang="less">
   @import "~@coderdojo/cd-common/common/_colors";
+  @import "../common/variables";
 
   .cd-dashboard-stats {
-    background-color: #fff;
+    background-color: #f4f5f6;
```

Comment:
Where does this value comes from ?
Don't we have an eq for that color available ?

Reply:
It's from the dojo details page - its the left column bar color